### PR TITLE
Implement/IEquippableItem interface

### PR DESCRIPTION
**Pull request by boscohyun**
- Add `IEquippableItem` interface.
- Implement 'IEquippableItem` from `Equipment` and `Costume`
- Bug fix in `Inventory.HasItem(Guid)`: Since this method is only being used for unit tests, I think it is a change that does not affect the chain.

### CODE REVIEW

File: Lib9c/Model/Item/Costume.cs [hunk 1]
```diff
@@ -8,12 +8,13 @@
 namespace Nekoyume.Model.Item
 {
     [Serializable]
-    public class Costume : ItemBase, INonFungibleItem
+    public class Costume : ItemBase, INonFungibleItem, IEquippableItem
     {
         public bool equipped = false;
```
Comment (ipdae) [inline Lib9c/Model/Item/Costume.cs:13]: this field can be private

Comment (boscohyun) [reply Lib9c/Model/Item/Costume.cs:13]: @ipdae We already use this field in `AvatarState`, `Sell2`, `Sell3`..😭

Comment (ipdae) [reply Lib9c/Model/Item/Costume.cs:13]: 😅 Could you please add a comment this field not to use anymore?

Comment (boscohyun) [reply Lib9c/Model/Item/Costume.cs:13]: And.. I find some week logic https://github.com/boscohyun/lib9c/blob/e3ea312f2f806672db7dc7fef5d112bb98e8d3c4/Lib9c/Model/State/AvatarState.cs#L522

**Review by ipdae — APPROVED** [review]

**Review by Unengine — APPROVED** [review]

**Pull request merged**